src/functions-reference/matrix_operations.Rmd

```diff
@@ -1237,66 +1237,270 @@ The cumulative sum of v
 `row_vector` **`cumulative_sum`**`(row_vector rv)`<br>\newline
 The cumulative sum of rv
 
-## Covariance Functions {#covariance}
+## Gaussian Process Covariance Functions {#gaussian-process-covariance-functions}
 
-### Exponentiated quadratic covariance function
+The Gaussian process covariance functions compute the covariance between a
+set of points and itself, $K_{ij} = k(x_i, x_j)$, or the covariance between
+two different sets of points, $K_{ij} = k(x_i, x_j)$. $x$ can be an array of
+scalars for a one dimensional kernel or an array of vectors for a
+multidimensional kernel.
```

Comment:
May be vector of doubles, instead of scalars. May be make x boldface so we know it's a vector?

Can you also indicate somewhere that  x_i is observation (row) i of x? 

Also, instead of the kernel being one-dimensional, the input data is multi-dimensional. Or the design matrix has more than one covariate. Something like that. We say the GP is 1-D, not the kernel.